File: android/app/src/main/java/me/pjq/rpicar/BaseApplication.java
package me.pjq.rpicar;

import android.app.Application;
import android.content.Context;

/**
 * Created by i329817(Jianqing.Peng@sap.com) on 18/11/2017.
 */

public class BaseApplication extends Application {
    static Context context;

    @Override
    public void onCreate() {
        super.onCreate();

        context = this;
    }

    public static Context getContext() {
        return context;
    }
}

File: android/app/src/main/java/me/pjq/rpicar/BaseApplication.kt
package me.pjq.rpicar

import android.app.Application
import android.content.Context

/**
 * Created by i329817(Jianqing.Peng@sap.com) on 18/11/2017.
 */

class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        context = this
    }

    companion object {
        lateinit var context: Context
            internal set
    }
}
